### ISSUE TITLE
National RADAR instance catalog/timeseries only has two pages of data

### ISSUE THREAD

**Issue by adamkorynta** [USACE/cwms-data-api#186]
The second page of data does not have a `next-page` cursor.

Initial query: 

```
https://cwms-data.usace.army.mil/cwms-data/catalog/timeseries
```

This give me back the next-page cursor `U0FNLzI0VEggU1QgQkhBTS5GTE9XLklOU1QuMC4wLlJBVy1VU0dTO1VOSVRTPUNNU3x8fDIwNDcxMg==` which I use in the follow-up query:

```
https://cwms-data.usace.army.mil/cwms-data/catalog/timeseries?page=U0FNLzI0VEggU1QgQkhBTS5GTE9XLklOU1QuMC4wLlJBVy1VU0dTO1VOSVRTPUNNU3x8fDIwNDcxMg==
```

This returned payload has no `next-page` cursor.

```

[response.txt](https://github.com/USACE/cwms-radar-api/files/9097562/response.txt)

**Comment by MikeNeilson** [comment]
I was able to recreate but it appears to only happen if the office is not specified. so likely an issue with those queries.

**Comment by MikeNeilson** [comment]
fixed in 2.4.5. query was rewritten.
